resources/istio/values.yaml

```diff
@@ -9,6 +9,7 @@ kyma:
   namespaces2Label:
     - istio-system
     - kube-system
+    - cert-manager
```

Comment:
Does it make sense? Job in istio will be executed before cert-manager is deployed. Namespace won't be labeled.

Reply:
It could be labelled as long as cert-manager component is installed before Istio. The job itself belongs to Istio, so the order in this file does not matter.

Reply:
I cannot agree. Introducing dependencies between components based on the order of installation is against any good practices. I spent last year fighting with dependencies and making parallel installation possible.

Reply:
I mean in this case I have not found any better way to deal with this to support old installer approach. Of course it is not a good practice to introduce any kind of dependencies. Here I simply reused what was in-place.